src/main/webapp/app/overview/exercise-chatbot/exercise-chatwidget/exercise-chat-widget.component.scss

```diff
@@ -0,0 +1,81 @@
+.container {
+    position: relative;
+}
+
+.chat-widget {
+    position: fixed;
+    bottom: 130px;
+    right: 50px;
+    width: 300px;
+    height: 400px;
+    background-color: white;
+    box-shadow: 0 0 50px rgba(0, 0, 0, 0.4);
+}
+
+.chat-header {
+    display: flex;
+    justify-content: space-between;
+    height: 50px;
+    background-color: #212529;
+    color: white;
+    font-weight: bold;
+    font-family: Arial;
```

Comment:
Another client style warning. Please address this:
> Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)

Also do you really need to use a special font family declaration here? At least for me it doesn't make much of a difference whether this is Arial or the default font.

Reply:
fixed